api/foreign.go

```diff
@@ -337,6 +409,41 @@ func (entity *ForeignActivitiesBenefits) Valid() (bool, error) {
 	return entity.List.Valid()
 }
 
+// ClearNos clears the "no" answers on application rejection
+func (entity *ForeignActivitiesBenefits) ClearNos() error {
+
+	entity.HasBenefits.ClearNo()
+
+	for _, benefitSetItem := range entity.List.Items {
+		benefitNames := []string{"ContinuingBenefit", "FutureBenefit", "OneTimeBenefit", "OtherBenefit"}
+
+		for _, benefitName := range benefitNames {
+
+			benefitItem, benefitErr := benefitSetItem.GetItemValue(benefitName)
+			if benefitErr != nil {
+				return errors.Wrap(benefitErr, fmt.Sprintf("error fetching benefit named %s from foreign activities", benefitName))
+			}
+
+			// Sometimes, OtherBenefit if it hasn't been selected seems to be sent as a textarea, but in that case we can't clear it
```

Comment:
this sounds like something we might want to look into so we at least know why it is happening. not sure where we're tracking work, but is this something it's worth adding a small chore or story for?

Reply:
I created [EN-3822]